src/cli/report.ts

```diff
@@ -1,12 +1,9 @@
 import * as ts from "typescript";
 
+export const prepareDiagnosticForFormatting = (diagnostic: ts.Diagnostic) =>
+    diagnostic.source === "typescript-to-lua" ? { ...diagnostic, code: "TL" as any } : diagnostic;
```

Comment:
can we not use code TSTL?

Reply:
`TS` is a prefix for all error codes, so by setting it to `TL` it becomes `TSTL`